config/_templates/dataset/yaak.yaml

```diff
@@ -20,9 +20,9 @@ sources:
     (@=source_id@):
       index_column: meta/ImageMetadata.(@=source_id@)/frame_idx
       source:
-        _target_: rbyte.io.FfmpegFrameSource
+        _target_: rbyte.io.TorchCodecFrameSource
         path: ${data_dir}/(@=input_id@)/(@=source_id@).pii.mp4
-        resize_shorter_side: 324
+        device: cpu
```

Comment:
also maybe keep `device: cuda` as a default value? we don't play games

Reply:
can do but this would require we make sure compatible ffmpeg is available on all nodes via fax

Reply:
we'd want the templates to be runnable in CI, so until we get a cuda runner it's cpu